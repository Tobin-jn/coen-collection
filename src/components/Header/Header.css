.Header {
  background-color: #e8e8e8;
  height: 120px;
}

.coen-collection {
  color: white;
  background: url("../../images/wood.jpg");
  font-size: 100px;
  font-weight: bold;
  text-align: center;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
}
