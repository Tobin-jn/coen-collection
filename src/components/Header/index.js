import React from 'react'
import './Header.css';

const Header = () => {
  return (
    <div className="Header">
      <h1>The Coen Collection</h1>
    </div>
  )
}

export default Header