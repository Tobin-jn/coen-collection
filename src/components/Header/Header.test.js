/* eslint-disable */

import React from "react";
import { shallow } from "enzyme";
import Header from "./index";

describe("Card", () => {
  it("Renders like snapshot", () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot();
  });
});