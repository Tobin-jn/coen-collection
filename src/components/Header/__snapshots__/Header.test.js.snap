// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Renders like snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="coen-collection"
      >
        coenCollection
      </h1>,
      "className": "Header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "coenCollection",
        "className": "coen-collection",
      },
      "ref": null,
      "rendered": "coenCollection",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="coen-collection"
        >
          coenCollection
        </h1>,
        "className": "Header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "coenCollection",
          "className": "coen-collection",
        },
        "ref": null,
        "rendered": "coenCollection",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
